common/main/java/com/couchbase/lite/internal/exec/SerialExecutor.java

```diff
@@ -60,6 +60,13 @@ class SerialExecutor implements ExecutionService.CloseableExecutor {
         this.executor = executor;
     }
 
+    /**
+     * Get the number of tasks awaiting execution.
+     *
+     * @return the number of tasks awaiting execution.
+     */
+    public int getPending() { return pendingTasks.size(); }
```

Comment:
Does it need to be called under `synchronized (this) { }` for thread-safety?

Reply:
I didn't think so.... but let me have another look.

Reply:
You are right: it does.   Fixed.